portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+
+// Assign emoji's hex code point to variable 
+const VIRGO_EMOJI = 0x264d;
+const TELEVISION_EMOJI=0x1f4fa;
+const OLD_WOMAN_EMOJI=0x1f475;
+
+/*
+ *Convert emoji to string
+ */
+function emojiString(emoji){
```

Comment:
Replace "emoji" with "emoji_code". Otherwise it's confusing. 

Also please annotate types according to: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

I won't block this PR tho. ill LGTM but make sure you annotate types in follow up PRs